draft-pauly-oblivious-doh.md

```diff
@@ -293,15 +292,39 @@ aead_id
 public_key
 : The HPKE public key used by the client to encrypt Oblivious DoH queries.
 
-# Oblivious DoH Message Format {#encryption}
+# Protocol Encoding {#encryption}
 
-There are two types of Oblivious DoH messages: Queries (0x01) and Responses (0x02). Both
-are encoded as follows:
+## Message Format
+
+There are two types of Oblivious DoH messages: Queries (0x01) and Responses (0x02).
+Both messages carry the following information:
+
+1. A DNS message, which is either a Query or Response, depending on context.
+1. Padding of arbitrary length which MUST contain all zeros.
+
+They are encoded encoded using the following structure:
```

Comment:
s/encoded encoded/encoded

(sorry @chris-wood, I still haven't figured out how to use suggestions :-)